lib/hooks/useFetchGithubPRInfo.ts

```diff
@@ -0,0 +1,42 @@
+import { Fetcher } from "swr";
+import useSWR from "swr";
+
+// Leaving this fetcher here as this solution will be changed in the future when the backend supports the github Pull information
```

Comment:
this fetcher will likely not be replaced but used side by side. it should not be marked as redundant or low effort, under any circumstance

Reply:
Can I call this `githubFetcher` separately in the libs?

Reply:
Separate from what?

Reply:
Like move it into a separate file that is exported and used anywhere

Reply:
no, that's why it's a hook, the fetcher itself could handle rate limiting and be put in lib, but first it has to be functional